src/modifiedsphericalbessel.jl

```diff
@@ -0,0 +1,33 @@
+
+function sphericalbesselk_int(v, x)
```

Comment:
maybe `v::Int`?

Reply:
ah, good call. Will do that in next commit.